Moblin/View/Settings/Watch/WatchSettingsView.swift

```diff
@@ -15,6 +15,19 @@ struct WatchSettingsView: View {
                     Text("Display")
                 }
             }
+            Section {
+                Toggle("Remote Control Info and Preview", isOn: Binding(get: {
+                    model.database.watch!.remoteControl!
+                }, set: { value in
+                    model.database.watch!.remoteControl = value
+                    model.store()
+                }))
+            } footer: {
+                Text("""
+                Display data and preview from the remote control streamer \
+                instead of from the device connected to the watch.
+                """)
```

Comment:
I think this should say something like "Use the watch as remote control assistant instead of ...", or at least mention the assistant, as it is the assistant functionality we expose in the watch.

Reply:
I have written a new text